packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/GetInheritedRulesTests.java

```diff
@@ -91,6 +96,37 @@ public void getInheritedRules()
                       .and().entriesListCountIs(1);
     }
 
+    /**
+     * Check we only get the owned rules and no inherited rules in the child folder when parent rule isn't inheritable.
+     */
+    @Test (groups = { TestGroup.REST_API, TestGroup.RULES, TestGroup.SANITY })
```

Comment:
Is this e2e correct for what's implemented? I think you should create a rule in the parent folder which has inherited set as false and then ensure that the rule is not inherited by the child. 

Also, why are you creating a rule in the child folder?